app/exec/extension/_lib/vsix-manifest-builder.ts

```diff
@@ -320,6 +320,11 @@ export class VsixManifestBuilder extends ManifestBuilder {
 					});
 				}
 				break;
+			case "githubflavoredmarkdown":
+				if(value !== "true" && value !== "false") {

```

Comment:
Why is the value expected to be a string? Why wouldn't it just be a regular boolean?